cmd/pullrequest-init/fake_github_test.go

```diff
@@ -41,7 +41,10 @@ func TestFakeGitHubComments(t *testing.T) {
 		t.Fatalf("List Issues: wanted [], got %+v, %+v, %v", got, resp, err)
 	}
 
-	if _, _, err := client.Issues.CreateComment(ctx, owner, repo, prNum, comment); err != nil {
+	comment, _, err := client.Issues.CreateComment(ctx, owner, repo, prNum, &github.IssueComment{
```

Comment:
i might just not be seeing it, but I can't find where the `comment` var was declared before this change :O Was this a bug in the tests? If so that would mean they aren't running in CI and there's no linting happening... 😨

Reply:
😨 😨 😨 😨 😨 😨 😨 😨